src/vellum/workflows/nodes/displayable/code_execution_node/utils.py

```diff
@@ -2,9 +2,20 @@
 from typing import Union
 
 
+def get_project_root() -> str:
```

Comment:
Could you elaborate on why these code changes were needed?

Reply:
I was getting a file not found for the given path, I suspect it was because we changed directory structures when we moved from old prototype repo to here